src/ui/server/controllers/insurance/business/nature-of-business/map-submitted-data/index.test.ts

```diff
@@ -0,0 +1,103 @@
+import { RequestBody } from '../../../../../../types';
+import { FIELD_IDS } from '../../../../../constants';
+import mapSubmittedData from '.';
+import { mockExporterBusiness } from '../../../../../test-mocks';
+import { stripCommas } from '../../../../../helpers/string';
+
+const { EXPORTER_BUSINESS } = FIELD_IDS.INSURANCE;
+const { GOODS_OR_SERVICES, YEARS_EXPORTING, EMPLOYEES_UK, EMPLOYEES_INTERNATIONAL } = EXPORTER_BUSINESS.NATURE_OF_YOUR_BUSINESS;
+
+describe('controllers/insurance/business/nature-of-business/map-submitted-data', () => {
+  describe('when all fields are provided and all number fields contain a comma', () => {
+    it('should return the formBody with the commas replaced', () => {
+      const mockBody = {
+        _csrf: '1234',
+        __typename: 'ExporterBusinessResponse',
```

Comment:
I don't think we need `__typename` anywhere in this file

Reply:
only there to check it is removed by mapping

Reply:
as it comes in the requestBody

Reply:
But the data `mapSubmittedData` will get is from the UI form body? That won't contain the `__typename`, this will only come from graphQL query response?

Reply:
Created a ticket do after this PR as it's in other places